Packages/Tracking/Examples~/Interaction Engine Examples/9. Hand Ray/Hand Ray.unity

```diff
@@ -0,0 +1,2829 @@
+%YAML 1.1
```

Comment:
This scene has missing references on both Directional Light and Fill Light

Reply:
could you resolve this thread if it is, please?